keps/sig-scheduling/5007-device-attach-before-pod-scheduled/kep.yaml

```diff
@@ -0,0 +1,46 @@
+title: DRA Device Attach Before Pod Scheduled
+kep-number: 5007
+authors:
+  - "@KobayashiD27"
+owning-sig: sig-scheduling
+#participating-sigs:
+#  - sig-aaa
+#  - sig-bbb
+status: implementable
+#|implemented|deferred|rejected|withdrawn|replaced
+creation-date: 2025-01-24
+reviewers:
+  - "@pohly"
+approvers:
+  - TBD
```

Comment:
@pohly 
I apologize for my unfamiliarity with the review process. Could you please let me know whose names I can list as approvers or reviewers? Is it acceptable to include either @sanposhiho or @alculquicondor?

Reply:
Would it be possible to pass the KEP freeze with just the approval of sig-scheduling, considering the scope of this KEP?

Reply:
I think it falls under SIG Scheduling and not SIG Node, because all of the changes are in the scheduler. Therefore approval from SIG Scheduling should be sufficient.

We still need to find a volunteer for the approver role. I can't decide that.

Reply:
@alculquicondor Am I eligible to be? Is `approver` on KEP limited to the SIG leads level, or allowed to include the approvers on k/k?

Reply:
No, only TLs. But please give a first pass and I'll have a look afterwards.